_posts/courses/earth-analytics-python/01-science-toolbox/2018-02-05-science-toolbox-landing.md

```diff
@@ -3,7 +3,6 @@ layout: single
 category: courses
 title: "Get Started with Data in Python / Jupyter Notebooks"
 permalink: /courses/earth-analytics-python/python-open-science-toolbox/
-modified: 2018-09-04
```

Comment:
this is weird @jlpalomino  i added these dates today.

Reply:
fixed. not sure how that happened

Reply:
@lwasser The changes look right, except that I don't see is an updated image for the homework 1 plot on the bottom of the landing page. The change was small - the title was updated to just 2013 (before it was 2003-2013).

Reply:
@jlpalomino  when you review a lesson you can comment on the line that is at issue. can you kindly do that please so i can see exactly what is missing and where?
it's weird that one of my edits got lost and one of yours is lost too... something is up with our workflow . let's try to review it together so we can sort out for the future.

i'm inclined at this point to work on master rather than a branch since the bootcamp is done. are you ok with that?

Reply:
@lwasser I definitely can add comment using specific line numbers in the future. It does not help in this case because the change I was referring to was in the plot image, not the text on the page.  Line 203 - I updated the image for the homework 1 plot with a new title. It looks like you resolved it with: https://github.com/earthlab/earthlab.github.io/pull/619/commits/6972f357343e3a62d97cac3e4cf8d40efa2cdb66

The bootcamp is done but I plan to make changes after you and I have had a chance to talk. Not sure if this makes a difference for your idea to work in master or not.